docs/relay/nodes.rst

```diff
@@ -52,7 +52,7 @@ Example of a custom node:
 
         @staticmethod
         def to_global_id(type, id):
-            return '{}:{}'.format(type, id)
+            return f'{type}:{id}'
```

Comment:
Can we get black to run on the examples in the docs as well? In this case the quotes should be double quotes.

Reply:
I've been trying to blacken them all, but clearly missed some. Lemme know if there are others. I'll do another run through.

Reply:
NM. This is addressed in https://github.com/graphql-python/graphene/pull/992